src/modules/checkout/ui/views/checkout-view.tsx

```
@@ -0,0 +1,79 @@
+  useEffect(() => {
+    if (error?.data?.code === 'NOT_FOUND') {
+      clearCart();
+      warning('Invalid products found, cart cleared');
+    }
+  }, [error, clearCart, warning]);
+
+  if (isLoading) {
+    return (
+      <div className="lg:pt-16 pt-4 px-4 lg:px-12">
+        <EmptyState type="loader" />;
+      </div>
```

Biome (1.9.4)

[error] There is a suspicious semicolon in the JSX element.

This is usually the result of a typo or some refactor gone wrong.
Remove the semicolon, or move it inside a JSX element.

(lint/suspicious/noSuspiciousSemicolonInJsx)